puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/test/testForkliftCentosDevel.groovy

```diff
@@ -0,0 +1,40 @@
+pipeline {
+    agent none
+
+    options {
+        timestamps()
+            timeout(time: 2, unit: 'HOURS')
+            disableConcurrentBuilds()
+            ansiColor('xterm')
+    }
+
+    stage('forklift centos7-devel pipeline tests') {
+        agent { label 'el' }
+        environment {
+            cico_job_name = "foreman-forklift-centos-devel-test"
+        }
+
+        steps {
+            git_clone_foreman_infra()
+
+            withCredentials([string(credentialsId: 'centos-jenkins', variable: 'PASSWORD')]) {
+                runPlaybook(
+                        playbook: 'ci/centos.org/ansible/jenkins_job.yml',
```

Comment:
Might want to get the indentations here right. EG: https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/katello/nightly.groovy#L52